docs/usage.rst

```diff
@@ -605,11 +605,11 @@ zhmc_partition_storage_groups                           D     G     Storage grou
 zhmc_storagegroup_type_int                              D     G     Storage group type as integer (0=fcp, 1=fc, 2=nvme, 99=unsupported value)
 zhmc_storagegroup_fulfillment_state_int                 D     G     Storage group fulfillment state as integer (0=complete, 1=pending, 2=pending-with-mismatches, 3=checking-migration, 4=incomplete, 99=unsupported value)
 zhmc_storagegroup_shared                                D     G     Boolean indicating whether the storage group is shared (0=false, 1=true)
-zhmc_storagegroup_max_partitions                        D     G     Maximum number of partitions a storage group can be attached to
+zhmc_storagegroup_max_partitions                        D     G     Maximum number of partitions a storage group can be attached to; only for FCP-type SGs
```

Comment:
Isn't this property also applicable for ficon and nvme storage groups ?

Reply:
Not according to the WS-API book.

Reply:
I also verified with an NVME SG on T224 and with a FC SG in one of our ZaaS DCs that the "max-partitions" property is not present on such SGs.